File: 94/Solution_94.py
class TreeNode:
    def __init__(self , val = 0 , left = None , right = None):
        self.val = val
        self.left = left
        self.right = right
    
def inorder_traversal(root):
    if not root:
        return []
    return inorder_traversal(root.left) + [root.val] + inorder_traversal(root.right)

root = TreeNode(1)
root.right = TreeNode(2)
root.right.left = TreeNode(3)

print(inorder_traversal(root))

File: 94/Solution_94.c
#include <stdio.h>
#include <stdlib.h>


typedef struct TreeNode {
    int value;
    struct TreeNode* left;
    struct TreeNode* right;
} Node;

Node* createNode(int value){
    Node* newNode = (Node*) malloc(sizeof(Node));
    newNode->value = value;
    newNode->left = NULL;
    newNode->right = NULL;
    return newNode;
}

void inorderTraversal(Node* root){
    if (root == NULL) return;
    inorderTraversal(root->left);
    printf("%d",root->value);
    inorderTraversal(root->right);

}

int main(){
    Node* root = createNode(1);
    root->right = createNode(2);
    root->right->left = createNode(3);

    printf("result = \n");
    inorderTraversal(root);
    printf("\n");

    return 0;

}
